CHANGES

```diff
@@ -48,6 +48,9 @@ Features added
 * #11572: Improve ``debug`` logging of reasons why files are detected as out of
   date.
   Patch by Eric Larson.
+* #11300: Improve ``SigElementFallbackTransform`` fallback logic and signature
+  text elements nodes. See :ref:`here <nodes>` for more details.
```

Comment:
Try to avoid links with the word "here" in general. Perhaps "See the documentation for more details" owtte